definition/data/sheets/CaseEventToFields.json

```diff
@@ -4,11 +4,13 @@
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ReviewedPaperResponse", "CaseFieldID": "scannedDocuments", "PageID": 1, "DisplayContext": "COMPLEX", "FieldShowCondition":"paperResponseType=\"BULK_SCAN\"", "PageFieldDisplayOrder": 2},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ReviewedPaperResponse", "CaseFieldID": "evidenceHandled", "PageID": 2, "DisplayContext": "MANDATORY", "FieldShowCondition":"paperResponseType=\"BULK_SCAN\"", "PageFieldDisplayOrder": 1},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "GenerateOrder", "CaseFieldID": "directionList", "PageID": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 1, "CallBackURLMidEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/mid", "RetriesTimeoutURLMidEvent": "5,10,15"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "GenerateOrder", "CaseFieldID": "extraDocUploadList", "PageID": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 2, "FieldShowCondition": "directionList CONTAINS\"DOCUMENTS\""},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "GenerateOrder", "CaseFieldID": "docUploadDeadline", "PageID": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 3, "FieldShowCondition": "directionList CONTAINS\"DOCUMENTS\""},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "GenerateOrder", "CaseFieldID": "eyewitnessUploadDeadline", "PageID": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 4, "FieldShowCondition": "directionList CONTAINS\"EYEWITNESS\""},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "GenerateOrder", "CaseFieldID": "eyewitnessUploadForParty", "PageID": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 5, "FieldShowCondition": "directionList CONTAINS\"EYEWITNESS\""},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "GenerateOrder", "CaseFieldID": "docUploadForParty", "PageID": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 6, "FieldShowCondition": "directionList CONTAINS\"DOCUMENTS\""},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "GenerateOrder", "CaseFieldID": "eyewitnessUploadDeadline", "PageID": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 2, "FieldShowCondition": "directionList CONTAINS\"EYEWITNESS\""},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "GenerateOrder", "CaseFieldID": "eyewitnessUploadForParty", "PageID": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 3, "FieldShowCondition": "directionList CONTAINS\"EYEWITNESS\""},
```

Comment:
shouldn't we update the `PageFieldDisplayOrder` for these above two fields?

Reply:
Missed deleting them, cleaned now